### PR TITLE
Prepare Lisk Core to use Lisk SDK 6.0.0 - Closes #582

### LINKED ISSUE #582: Prepare Lisk Core to use Lisk SDK 6.0.0
### Description

Update Lisk Core to use latest Lisk SDK

- Update Lisk SDK to use canary version e.g `5.2.2-alpha.31164`
- Update application instantiation
- Remove HTTP API Plugin
- Update `devnet, alphabet` genesis block
- Update `devnet, alphabet` application config (generator, naming changes, etc..)

### Motivation

Prepare Lisk Core to kick start sapphire phase development

### Additional Information

- Canary build information https://npm.lisk.com/-/web/detail/lisk-sdk
- We can use Lisk SDK genesis block create function to generate `devnet, alphabet` genesis block

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?

This PR resolves #582

### How was it solved?

- Updated Lisk SDK to use canary version e.g 5.2.2-alpha.31164
- Updated application instantiation
- Removed HTTP API Plugin
- Updated devnet, alphabet genesis block
- Updated devnet, alphabet application config (generator, naming changes, etc..)

### How was it tested?

- npm install --registry https://npm.lisk.com
- npm run build
- ./bin/run start -n devnet

### CODE REVIEW

File: src/base_ipc.ts [hunk 1]
```diff
@@ -166,29 +164,31 @@ export default abstract class BaseIPCCommand extends Command {
...
 	}
 
 	protected transactionToJSON(transaction: Record<string, unknown>): Record<string, unknown> {
 		if (this._client) {
 			return this._client.transaction.toJSON(transaction);
 		}
-		const assetSchema = this.getAssetSchema(
+		const commandSchema = this.getCommandSchema(
 			transaction.moduleID as number,
 			transaction.assetID as number,
```
Comment (nagdahimanshu) [inline src/base_ipc.ts:183]: Shouldn't it be?
```suggestion
			transaction.commandID as number,
```

File: config/alphanet/config.json [hunk 2]
```diff
@@ -41,9 +30,25 @@
...
+	"genesis": {
+		"blockTime": 10,
+		"communityIdentifier": "Lisk",
+		"maxTransactionsSize": 15360,
+		"minFeePerByte": 1000,
+		"modules": {},
+		"baseFees": [
+			{
+				"moduleID": 5,
+				"assetID": 0,
```
Comment (nagdahimanshu) [inline config/alphanet/config.json:44]:
```suggestion
				"CommandID": 0,
```

File: config/devnet/config.json [hunk 3]
```diff
@@ -41,10 +30,26 @@
...
+	"genesis": {
+		"blockTime": 10,
+		"communityIdentifier": "Lisk",
+		"maxTransactionsSize": 15360,
+		"minFeePerByte": 1000,
+		"modules": {},
+		"baseFees": [
+			{
+				"moduleID": 5,
+				"assetID": 0,
```
Comment (nagdahimanshu) [inline config/devnet/config.json:44]: Same for this:
```suggestion
				"commandID": 0,
```

**Review by nagdahimanshu — CHANGES_REQUESTED** [review]
Don't we need to update the relevant `md` files in the same issue? (For example, we removed `HTTP API Plugin` but it still exists in the options for the start command `lisk-core/docs/commands/start.md`)

**Comment by sameersubudhi** [comment]
@ManuGowda The following documents still require cleanup concerning the `--enable-http-api-plugin`, `--http-api-plugin-host` and `--http-api-plugin-port` flags. Please search for `http-api` throughout the codebase to find them all (12 occurrences in 4 files listed below).

> - README.md
> - docker/README.md
> - docs/commands/start.md
> - test/scripts/README.md

Similarly for `assetID` mentions in the following files:
> - config/betanet/config.json
> - config/mainnet/config.json
> - config/testnet/config.json
> - docs/commands/transaction.md
> - test/scripts/utils/transactions/create.ts

**Review by sameersubudhi — APPROVED** [review]

**Review by nagdahimanshu — APPROVED** [review]
LGTM 👍

**Review by shuse2 — APPROVED** [review]

**Pull request merged**